# Translations template for nova.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the nova project.
#
# Translators:
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: nova 14.0.0.0b3.dev750\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-08-25 06:15+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2016-01-20 01:49+0000\n"
"Last-Translator: Andreas Jaeger <jaegerandi@gmail.com>\n"
"Language: tr-TR\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Turkish (Turkey)\n"

msgid "\"Look for the VDIs failed"
msgstr "\"VDI arama başarısız"

#, python-format
msgid ""
"%(base)s virtual size %(base_size)s larger than flavor root disk size "
"%(size)s"
msgstr ""
"%(base)s sanal boyutu %(base_size)s nitelik kök diski boyutu %(size)s den "
"büyük"

#, python-format
msgid ""
"%(scheme)s is registered as a module twice. %(module_name)s is not being "
"used."
msgstr ""
"%(scheme)s bir modül olarak iki kere kaydedilmiş. %(module_name)s "
"kullanılmıyor."

#, python-format
msgid ""
"Aggregate %(aggregate_id)s: unrecoverable state during operation on %(host)s"
msgstr ""
"Takım %(aggregate_id)s: %(host)s istemcisi üzerindeki işlem sırasında "
"kurtarılamayacak durum"

msgid "An error occurred while refreshing the network cache."
msgstr "Ağ zulası tazelenirken bir hata oluştu."

#, python-format
msgid "Attaching PCI devices %(dev)s to %(dom)s failed."
msgstr "%(dev)s PCI aygıtlarının %(dom)s e eklenmesi başarısız."

msgid "Cannot cleanup migration files"
msgstr "Göç dosyaları temizlenemiyor"

#, python-format
msgid "Cannot reboot instance: %s"
msgstr "Sunucu yeniden başlatılamıyor: %s"

#, python-format
msgid "Caught error: %s"
msgstr "Yakalama hatası: %s"

msgid "Compute driver option required, but not specified"
msgstr "Hesaplama sürücüsü seçeneği gerekli, ama belirtilmemiş"

#, python-format
msgid "Connection to libvirt failed: %s"
msgstr "Libvirt bağlantısı başarısız: %s"

#, python-format
msgid "Could not bind to %(host)s:%(port)s"
msgstr "%(host)s:%(port)s a bağlanılamadı"

msgid "Could not decode scheduler options"
msgstr "Zamanlayıcı seçenekleri kodu çözülemedi"

#, python-format
msgid "Could not remove tmpdir: %s"
msgstr "tmpdir silinemedi: %s"

#, python-format
msgid "Could not stat scheduler options file %(filename)s"
msgstr "Zamanlayıcı seçenekleri dosyası %(filename)s durumu alınamadı"

msgid "Couldn't communicate with any cells"
msgstr "Hiçbir hücreyle iletişim kurulamadı"

#, python-format
msgid "Couldn't communicate with cell '%s'"
msgstr "Hücre '%s' ile iletişim kurulamadı"

#, python-format
msgid "Couldn't lookup app: %s"
msgstr "Uygulama aranamadı: %s"

#, python-format
msgid "Couldn't unmount the GlusterFS share %s"
msgstr "%s GlusterFS paylaşımı ayrılamıyor"

#, python-format
msgid "Couldn't unmount the NFS share %s"
msgstr "%s NFS paylaşımı ayrılamıyor"

#, python-format
msgid "Couldn't unmount the Quobyte Volume at %s"
msgstr "%s deki Quobyte Mantıksal Sürücüsü ayrılamıyor"

#, python-format
msgid "Couldn't unmount the share %s"
msgstr "%s paylaşımı ayrılamadı"

#, python-format
msgid "Creating config drive failed with error: %s"
msgstr "Yapılandırma sürücüsü oluşturma şu hatayla başarısız oldu: %s"

msgid "DB error"
msgstr "DB hatası"

msgid "Destroy VM failed"
msgstr "VM silme başarısız"

msgid "Destroy instance failed"
msgstr "Sunucu silme başarısız"

#, python-format
msgid "Detaching network adapter failed. Exception: %s"
msgstr "Ağ bağdaştırıcısı ayırma başarısız. İstisna: %s"

#, python-format
msgid ""
"Download handler '%(handler)s' raised an exception, falling back to default "
"handler '%(default_handler)s'"
msgstr ""
"İndirme işleyicisi '%(handler)s' bir istisna yükseltti, varsayılan "
"işleyiciye '%(default_handler)s' dönülüyor"

msgid "Download image error"
msgstr "İmaj indirme hatası"

#, python-format
msgid "Driver failed to attach volume %(volume_id)s at %(mountpoint)s"
msgstr ""
"Sürücü %(volume_id)s mantıksal sürücüsünü %(mountpoint)s noktasına ekleyemedi"

msgid "Environment variable 'NETWORK_ID' must be set."
msgstr "Çevresel değişken 'NETWORK_ID' ayarlanmış olmalı."

#, python-format
msgid "Error defining a domain with XML: %s"
msgstr "XML ile bir alanın tanımlanmasında hata: %s"

#, python-format
msgid "Error defining a secret with XML: %s"
msgstr "Bir sırın XML ile tanımlanması başarısız: %s"

#, python-format
msgid "Error deleting conntrack entries for %s"
msgstr "%s için conntrack girdileri silinirken hata"

#, python-format
msgid "Error deploying instance %(instance)s on baremetal node %(node)s."
msgstr ""
"%(instance)s sunucusunun çıplak metal %(node)s düğümü üzerine açılmasında "
"hata."

#, python-format
msgid "Error enabling hairpin mode with XML: %s"
msgstr "XML ile hairpin kipinin etkinleştirilmesinde hata: %s"

#, python-format
msgid "Error from last host: %(last_host)s (node %(last_node)s): %(exc)s"
msgstr "Son istemciden hata: %(last_host)s (node %(last_node)s): %(exc)s"

#, python-format
msgid "Error from libvirt during destroy. Code=%(errcode)s Error=%(e)s"
msgstr "Silme sırasında libvirt'den hata. Kod=%(errcode)s Hata=%(e)s"

#, python-format
msgid "Error from libvirt during undefine. Code=%(errcode)s Error=%(e)s"
msgstr ""
"Tanımlama kaldırma sırasında libvirt hatası. Kod=%(errcode)s Hata=%(e)s"

#, python-format
msgid "Error from libvirt during unfilter. Code=%(errcode)s Error=%(e)s"
msgstr "Filtre kaldırma sırasında libvirt hatası. Kod=%(errcode)s Hata=%(e)s"

#, python-format
msgid "Error injecting data into image %(img_id)s (%(e)s)"
msgstr "Verinin %(img_id)s imajına eklenmesinde hata (%(e)s)"

#, python-format
msgid "Error instantiating %(provider)s: %(exception)s"
msgstr "%(provider)s ilklendirmede hata: %(exception)s"

#, python-format
msgid "Error launching a defined domain with XML: %s"
msgstr "Tanımlanmış bir alanın XML ile başlatılmasında hata: %s"

msgid "Error locating next hop for message"
msgstr "İleti için sonraki atlayış noktası tespit edilirken hata"

msgid "Error locating next hops for message"
msgstr "İleti için sonraki noktalar tespit edilirken hata"

msgid ""
"Error occurred during volume_snapshot_create, sending error status to Cinder."
msgstr ""
"volume_snapshot_create sırasında hata oluştu, hata durumu Cinder'e "
"gönderiliyor."

msgid ""
"Error occurred during volume_snapshot_delete, sending error status to Cinder."
msgstr ""
"volume_snapshot_delete sırasında hata oluştu, hata durumu Cinder'e "
"gönderiliyor."

#, python-format
msgid "Error on '%(path)s' while checking direct I/O: '%(ex)s'"
msgstr "Direk I/O kontrol edilirken '%(path)s' üzerinde hata: '%(ex)s'"

#, python-format
msgid ""
"Error preparing deploy for instance %(instance)s on baremetal node %(node)s."
msgstr ""
"%(instance)s sunucusunun %(node)s çıplak metal düğümü üzerinde açılmasında "
"hata."

msgid "Error processing message locally"
msgstr "İleti yerel olarak işlenirken hata"

#, python-format
msgid "Error releasing DHCP for IP %(address)s with MAC %(mac_address)s"
msgstr ""
"%(mac_address)s MAC'ine sahip %(address)s IP'si için DHCP serbest "
"bırakılamadı"

#, python-format
msgid "Error scheduling instances %(instance_uuids)s"
msgstr "Sunucular %(instance_uuids)s eşzamanlanırken hata"

msgid "Error sending message to next hops."
msgstr "İleti sonraki noktalara gönderilirken hata."

#, python-format
msgid "Error starting xvp: %s"
msgstr "XVP başlatılırken hata oluştu: %s"

msgid "Error trying to Rescue Instance"
msgstr "Sunucu Kurtarılmaya çalışılırken hata"

msgid "Error trying to reschedule"
msgstr "Yeniden zamanlama yapmaya çalışılırken hata"

msgid "Error waiting for responses from neighbor cells"
msgstr "Komşu hücrelerden yanıt beklerken hata"

#, python-format
msgid "Error while trying to clean up image %s"
msgstr "%s imajı temizlenmeye çalışılırken hata"

#, python-format
msgid "Error: %s"
msgstr "Hata: %s"

#, python-format
msgid "Exception dispatching event %(event)s: %(ex)s"
msgstr "%(event)s olayı dağıtılırken istisna: %(ex)s"

#, python-format
msgid "Exception handling resource: %s"
msgstr "İstisna ele alma kaynağı: %s"

msgid "Exception in string format operation"
msgstr "Karakter dizisi biçimleme işleminde istisna"

msgid "Exception loading extension"
msgstr "Eklenti yüklenirken istisna"

#, python-format
msgid "Exception running %(name)s %(type)s-hook: %(obj)s"
msgstr "%(name)s %(type)s-hook çalıştırılırken istisna: %(obj)s"

msgid "Exception while creating table."
msgstr "Tablo oluştururken istisna."

#, python-format
msgid "Exception while waiting completion of volume snapshots: %s"
msgstr "Mantıksal sürücü anlık görüntülerinin bitmesini beklerken istisna: %s"

#, python-format
msgid "Extending virtual disk failed with error: %s"
msgstr "Sanal diskin büyütülmesi şu hatayla başarısız oldu: %s"

#, python-format
msgid "Failed removing bridge device: '%s'"
msgstr "Köprü aygıtı kaldırma başarısız: '%s'"

#, python-format
msgid "Failed removing net device: '%s'"
msgstr "Ağ aygıtı kaldırma başarısız: '%s'"

msgid "Failed storing info cache"
msgstr "Bilgi zulası depolanması başarısız"

msgid ""
"Failed to allocate PCI devices for instance. Unassigning devices back to "
"pools. This should not happen, since the scheduler should have accurate "
"information, and allocation during claims is controlled via a hold on the "
"compute node semaphore"
msgstr ""
"Sunucu için PCI aygıtları ayırma başarısız. Aygıtlar havuza geri "
"bırakılıyor. Zamanlayıcı doğru bilgiye sahip olduğundan ve talepler "
"sırasında ayırma, hesaplama düğümü semaforundaki tutma tarafından kontrol "
"edildiğinden bu olmamalı"

msgid "Failed to allocate network(s)"
msgstr "Ağ(lar) ayrılamadı"

#, python-format
msgid "Failed to attach %(volume_id)s at %(mountpoint)s"
msgstr "%(volume_id)s nin %(mountpoint)s e eklenmesi başarısız"

#, python-format
msgid "Failed to attach volume at mountpoint: %s"
msgstr "Mantıksal sürücü bağlama noktasına eklenemedi: %s"

#, python-format
msgid "Failed to change vm state of %(instance_name)s to %(req_state)s"
msgstr ""
"%(instance_name)s vm durumunun %(req_state)s olarak değiştirilmesi başarısız"

msgid "Failed to check if instance shared"
msgstr "Sunucunun paylaşılıp paylaşılmadığı kontrolü yapılamadı"

#, python-format
msgid "Failed to cleanup directory %(target)s: %(e)s"
msgstr "%(target)s dizini temizleme başarısız: %(e)s"

msgid "Failed to cleanup snapshot."
msgstr "Anlık görüntü temizliği başarısız."

#, python-format
msgid "Failed to commit reservations %s"
msgstr "Rezervasyonlar gönderilemedi %s"

msgid "Failed to complete a deletion"
msgstr "Silme tamamlanamadı"

#, python-format
msgid "Failed to connect to volume %(volume_id)s with volume at %(mountpoint)s"
msgstr ""
"%(volume_id)s mantıksal sürücüsüne %(mountpoint)s noktasındaki mantıksal "
"sürücü ile bağlanma başarısız"

#, python-format
msgid ""
"Failed to copy cached image %(source)s to %(dest)s for resize: %(error)s"
msgstr ""
"Zuladaki imaj %(source)s %(dest)s e yeniden boyutlandırma için "
"kopyalanamadı: %(error)s"

msgid "Failed to deallocate networks"
msgstr "Ağ ilişikleri kesilemedi"

#, python-format
msgid "Failed to decrypt %(image_location)s to %(image_path)s"
msgstr "%(image_path)s için %(image_location)s çözümü başarısız"

#, python-format
msgid "Failed to destroy instance: %s"
msgstr "Sunucu silme başarısız: %s"

#, python-format
msgid "Failed to detach %(device_name)s. Exception: %(exc)s"
msgstr "%(device_name)s ayrılamadı: İstisna: %(exc)s"

#, python-format
msgid "Failed to detach volume %(volume_id)s from %(mp)s"
msgstr "%(volume_id)s mantıksal sürücüsünün %(mp)s den ayrılması başarısız"

#, python-format
msgid "Failed to download %(image_location)s to %(image_path)s"
msgstr "%(image_path)si için %(image_location)s indirme işlemi başarısız"

msgid "Failed to fetch glance image"
msgstr "Glance imajı getirilirken hata"

#, python-format
msgid "Failed to find an SR for volume %s"
msgstr "%s mantıksal sürücüsü için SR bulunamadı"

#, python-format
msgid "Failed to forget the SR for volume %s"
msgstr "%s mantıksal sürücüsü için SR unutma başarısız"

#, python-format
msgid "Failed to generate usage audit for instance on host %s"
msgstr "%s istemcisi üzerindeki sunucu için kullanım denetimi üretilemedi"

#, python-format
msgid "Failed to get compute_info for %s"
msgstr "%s için compute_info alınamadı"

msgid "Failed to get default networks"
msgstr "Varsayılan ağlar alınamadı"

#, python-format
msgid "Failed to get metadata for instance id: %s"
msgstr "Sunucu kimliği için metadata alınmada hata: %s"

msgid "Failed to get nw_info"
msgstr "nw_info alınamadı"

#, python-format
msgid "Failed to instantiate the download handler for %(scheme)s"
msgstr "%(scheme)s için indirme işleyicisi ilklendirilemedi"

#, python-format
msgid "Failed to load %(cfgfile)s: %(ex)s"
msgstr "%(cfgfile)s yüklenemedi: %(ex)s"

#, python-format
msgid "Failed to load %s"
msgstr "%s yüklenemedi"

#, python-format
msgid "Failed to load %s-api"
msgstr "%s-api yükleme başarısı"

#, python-format
msgid ""
"Failed to mount container filesystem '%(image)s' on '%(target)s': %(errors)s"
msgstr ""
"Kap dosya sistemi '%(image)s' '%(target)s' üstüne bağlanamadı: %(errors)s"

msgid "Failed to notify cells of BDM destroy."
msgstr "BDM silme hücrelerini bilgilendirme başarısız."

msgid "Failed to notify cells of BDM update/create."
msgstr "BDM güncelleme/oluşturma hücrelerini bilgilendirme başarısız."

msgid "Failed to notify cells of bw_usage update"
msgstr "Hücrelerin bw_usage güncellemesi ile bilgilendirilmeleri başarısız"

msgid "Failed to notify cells of instance fault"
msgstr "Hücreler sunucu arızasıyla ilgili bilgilendirilemedi"

msgid "Failed to notify cells of instance info cache update"
msgstr "Sunucu bilgi zula güncelleme hücrelerini bilgilendirme başarısız"

#, python-format
msgid "Failed to remove %(base_file)s, error was %(error)s"
msgstr "%(base_file)s dosyasını kaldırma başarısız, %(error)s"

#, python-format
msgid "Failed to request Ironic to provision instance %(inst)s: %(reason)s"
msgstr "Ironic'den %(inst)s sunucusunu açmasını isteme başarısız: %(reason)s"

#, python-format
msgid ""
"Failed to retrieve encryption metadata for volume %(volume_id)s: "
"%(exception)s"
msgstr "%(volume_id)s için şifreleme metadata'sı alma başarısız: %(exception)s"

msgid "Failed to retrieve ephemeral encryption key"
msgstr "Geçici şifreleme anahtarı alınması başarısız"

msgid "Failed to revert crashed migration"
msgstr "Çökmüş göçün geri alınması başarısız"

#, python-format
msgid "Failed to roll back reservations %s"
msgstr "Rezervasyonlar geri alınamadı %s"

msgid "Failed to rollback quota for failed finish_resize"
msgstr "Başarısız olmuş finish_resize için kota geri alma başarısız"

#, python-format
msgid "Failed to send message to cell: %(next_hop)s"
msgstr "İleti hücreye gönderilemedi: %(next_hop)s"

msgid "Failed to send state update notification"
msgstr "Durum güncelleme bildirimi gönderilemedi"

msgid "Failed to send updated snapshot status to volume service."
msgstr ""
"Güncellenmiş anlık görüntü durumunun mantıksal sürücü servisine gönderilmesi "
"başarısız."

msgid "Failed to set instance name using multi_instance_display_name_template."
msgstr ""
"multi_instance_display_name_template kullanarak sunucu ismi ayarlanamadı."

#, python-format
msgid "Failed to start %(name)s on %(host)s:%(port)s with SSL support"
msgstr "%(host)s:%(port)s üzerinde SSL destekli %(name)s başlatılamadı"

msgid "Failed to start instance"
msgstr "Sunucu başlatma başarısız"

msgid "Failed to stop instance"
msgstr "Sunucu durdurma başarısız"

#, python-format
msgid "Failed to swap volume %(old_volume_id)s for %(new_volume_id)s"
msgstr "%(old_volume_id)s nin %(new_volume_id)s ile yer değişmesi başarısız"

msgid "Failed to teardown container filesystem"
msgstr "Kap dosya sistemi koparılamadı"

msgid "Failed to umount container filesystem"
msgstr "Kap dosya sistemi ayırma başarısız"

msgid "Failed to unpause instance"
msgstr "Sunucu sürdürme başarısız"

#, python-format
msgid "Failed to untar %(image_location)s to %(image_path)s"
msgstr "%(image_path)s için %(image_location)s çıkarma işlemi başarısız"

msgid "Failed to update usages bulk deallocating floating IP"
msgstr "Kullanımlar güncellenemedi değişken IP toptan bırakılıyor"

msgid "Failed to update usages deallocating fixed IP"
msgstr "Kullanımlar güncellenemedi sabit IP bırakılıyor"

msgid "Failed to update usages deallocating floating IP"
msgstr "Kullanımlar güncellenemedi değişken IP bırakılıyor"

msgid "Failed to update usages deallocating network."
msgstr "Kullanımların güncellenmesi başarısız ağ bırakılıyor."

msgid "Failed to update usages deallocating security group"
msgstr "Kullanımların güncellenmesi başarısız güvenlik grubu bırakılıyor"

msgid "Failed to update usages deallocating server group"
msgstr "Kullanımların güncellenmesi başarısız sunucu grubu bırakılıyor."

#, python-format
msgid "Failed to upload %(image_location)s to %(image_path)s"
msgstr "%(image_path)s için %(image_location)s yükleme işlemi başarısız"

msgid "Failed to write configuration file"
msgstr "Yapılandırma dosyası yazma başarısız"

msgid "Failed to write inbound.csr"
msgstr "inbound.csr yazılamadı"

msgid "Failed while plugging vif"
msgstr "Vif takılırken başarısız olundu"

msgid "Failed while unplugging vif"
msgstr "Vif ayrılırken başarısız olundu"

msgid "Failure prepping block device"
msgstr "Blok aygıt hazırlanırken hata"

msgid "Failure while cleaning up attached VDIs"
msgstr "Ekli VDI'ler temizlenmesi başarısız"

#, python-format
msgid "Fatal Exception running %(name)s %(type)s-hook: %(obj)s"
msgstr "%(name)s %(type)s-hook çalıştırılırken ölümcül istisna: %(obj)s"

msgid "Guest does not have a console available"
msgstr "Konuğun kullanılabilir bir konsolu yok"

msgid "Host is member of a pool, but DB says otherwise"
msgstr "İstemci bir havuza üye, ama DB aksini söylüyor"

#, python-format
msgid ""
"Hostname has changed from %(old)s to %(new)s. A restart is required to take "
"effect."
msgstr ""
"Makine adı %(old)s den %(new)s e değişti. Etkinleşmesi için yeniden başlatma "
"gerekiyor."

#, python-format
msgid "Image size %(size)d exceeded flavor allowed size %(allowed_size)d"
msgstr ""
"İmaj boyutu %(size)d nitelik izin verilen boyutu %(allowed_size)d yi geçti"

#, python-format
msgid ""
"Instance compute service state on %s expected to be down, but it was up."
msgstr ""
"%s üzerindeki sunucu hesap servisi durumu kapalı olarak bekleniyordu, ama "
"açık."

msgid "Instance failed block device setup"
msgstr "Örnek blok aygıtı kurulumu hatalı"

#, python-format
msgid "Instance failed network setup after %(attempts)d attempt(s)"
msgstr "Sunucu %(attempts)d girişimden sonra ağ kurulumunda başarısız oldu"

msgid "Instance failed to spawn"
msgstr "Örnek oluşturmada hata"

#, python-format
msgid "Interface %s not found"
msgstr "Arayüz %s bulunamadı"

#, python-format
msgid "Invalid server_string: %s"
msgstr "Geçersiz server_string: %s"

#, python-format
msgid "Live Migration failure: %s"
msgstr "Canlı Göç başarısız: %s"

msgid "Migrate Receive failed"
msgstr "Göç Alma başarısız"

msgid "Migrate Send failed"
msgstr "Göç Gönderme başarısız"

#, python-format
msgid "Migration %s is not found during confirmation"
msgstr "Onaylama sırasında göç %s bulunamadı"

#, python-format
msgid ""
"Migration of instance %(instance_id)s to host %(dest)s unexpectedly failed."
msgstr ""
"%(instance_id)s sunucusunun %(dest)s istemcisine göçü beklenmedik şekilde "
"başarısız oldu."

msgid "Migration operation has aborted"
msgstr "Göç işlemi iptal edildi"

#, python-format
msgid "NBD error - previous umount did not cleanup /var/lock/qemu-nbd-%s."
msgstr "NBD hatası - önce umount /var/lock/qemu-nbd-%s yolunu temizlemedi."

#, python-format
msgid ""
"NOT IMPLEMENTED: The call to %(method)s is not supported by the agent. args="
"%(args)r"
msgstr ""
"UYGULANMAMIŞ: %(method)s çağrısı ajan tarafından desteklenmiyor. args="
"%(args)r"

msgid "Network driver option required, but not specified"
msgstr "Ağ sürücüsü seçeneği gerekiyor, ama belirtilmemiş"

#, python-format
msgid "Neutron Error adding rules to security group %s"
msgstr "%s güvenlik grubuna kural eklerken Neutron Hatası"

#, python-format
msgid "Neutron Error creating security group %s"
msgstr "Güvenlik grubu %s oluştururken Neutron Hatası"

#, python-format
msgid "Neutron Error getting security group %s"
msgstr "%s güvenlik grubu alınırken Neutron Hatası"

msgid "Neutron Error getting security groups"
msgstr "Güvenlik gruplarını alırken Neutron Hatası"

#, python-format
msgid "Neutron Error unable to delete %s"
msgstr "Neutron Hatası %s silinemiyor"

#, python-format
msgid "Neutron Error updating security group %s"
msgstr "%s güvenlik grubu güncellemesinde Neutron Hatası"

msgid "Neutron Error:"
msgstr "Neutron Hatası:"

#, python-format
msgid "Neutron Error: %s"
msgstr "Neutron Hatası: %s"

#, python-format
msgid "Neutron Reported failure on event %(event)s for instance %(uuid)s"
msgstr ""
"Neutron %(uuid)s sunucusu için %(event)s olayında başarısızlık raporladı"

#, python-format
msgid "Neutron error creating port on network %s"
msgstr "%s ağında bağlantı noktası oluştururken Neutron hatası"

#, python-format
msgid "No compute node record for host %s"
msgstr "%s istemcisi için hesap düğüm kaydı yok"

#, python-format
msgid "No save handler for %s"
msgstr "%s için kaydetme işleyicisi yok"

msgid "Not all properties needed are implemented in the compute driver"
msgstr "İhtiyaç duyulan tüm özellikler hesap sürücüsünde uygulanmamış"

msgid ""
"Periodic sync_power_state task had an error while processing an instance."
msgstr "Dönemsel sync_power_state görevi bir sunucuyu işlerken hata aldı."

msgid "Periodic task failed to offload instance."
msgstr "Dönemsel görev sunucu yükünü almada başarısız."

msgid "Plugin config_file get_val failed"
msgstr "Eklenti config_file get_val başarısız"

#, python-format
msgid "Pool-Join failed: %s"
msgstr "Havuz-Katılma başarısız: %s"

#, python-format
msgid "Pool-eject failed: %s"
msgstr "Havuz-çıkarma başarısız: %s"

#, python-format
msgid "Pool-set_name_label failed: %s"
msgstr "Havuz-set_name_label başarısız: %s"

#, python-format
msgid "Pre live migration failed at %s"
msgstr "Canlı öncesi göç %s de başarısız"

msgid "Read/Write data failed"
msgstr "Veri okuma/yazma başarısız"

#, python-format
msgid "Requested VM Generation %s, but provided VHD instead of VHDX."
msgstr "VM Oluşturma %s istendi, ama VHDX yerine VHD sağlandı."

#, python-format
msgid "Returning exception %s to caller"
msgstr "Çağırana %s istisnası döndürülüyor"

msgid "Service error occurred during cleanup_host"
msgstr "cleanup_host sırasında servis hatası oluştu"

msgid "Setting instance vm_state to ERROR"
msgstr "Sunucu vm_state HATA olarak ayarlanıyor"

msgid "Shutting down VM (cleanly) failed."
msgstr "VM kapatma (temizce) başarısız."

msgid "Shutting down VM (hard) failed"
msgstr "VM kapatma (katı) başarısız"

msgid ""
"Something wrong happened when trying to delete snapshot from shelved "
"instance."
msgstr ""
"Askıdaki sunucudan anlık görüntüler silinmeye çalışılırken bir şeyler ters "
"gitti."

#, python-format
msgid "TIMEOUT: The call to %(method)s timed out. args=%(args)r"
msgstr "ZAMAN AŞIMI: %(method)s çağrısı zaman aşımına uğradı. args=%(args)r"

#, python-format
msgid "Temporary directory is invalid: %s"
msgstr "Geçici dizin geçersiz: %s"

#, python-format
msgid "The Quobyte volume at %s is still in use."
msgstr "%s deki Quobyte mantıksal sürücüsü hala kullanımda."

#, python-format
msgid ""
"The agent call to %(method)s returned an invalid response: %(ret)r. args="
"%(args)r"
msgstr ""
"%(method)s ajan çağrısı geçersiz bir yanıt döndürdü: %(ret)r. args=%(args)r"

#, python-format
msgid "The call to %(method)s returned an error: %(e)s."
msgstr "%(method)s çağrısı hata döndürdü: %(e)s."

#, python-format
msgid "The call to %(method)s returned an error: %(e)s. args=%(args)r"
msgstr "%(method)s çağrısı bir hata döndürdü: %(e)s. args=%(args)r"

msgid "Transfer data failed"
msgstr "Veri aktarımı başarısız"

#, python-format
msgid "Unable to access floating IP %s"
msgstr "%s değişken IP'sine erişilemiyor"

msgid "Unable to access the rescue disk"
msgstr "Kurtarma diskine erişilemiyor"

#, python-format
msgid "Unable to attach volume to instance %s"
msgstr "Mantıksal sürücü %s sunucusuna eklenemiyor"

#, python-format
msgid "Unable to clear device ID for port '%s'"
msgstr "'%s' bağlantı noktası için aygıt kimliği temizlenemedi"

msgid "Unable to create VM snapshot, failing volume_snapshot operation."
msgstr "VM anlık görüntüsü oluşturulamıyor, volume_snapshot işlemi başarısız."

msgid ""
"Unable to create quiesced VM snapshot, attempting again with quiescing "
"disabled."
msgstr ""
"Susturulmuş VM anlık görüntüsü oluşturulamadı, susturma kapatılmış olarak "
"tekrar deneniyor."

msgid "Unable to destroy VBD"
msgstr "VBD silinemedi"

#, python-format
msgid "Unable to execute %(cmd)s. Exception: %(exception)s"
msgstr "%(cmd)s çalıştırılamadı. Hata: %(exception)s"

msgid "Unable to find SR from VBD"
msgstr "VBD'den SR bulunamadı"

msgid "Unable to find SR from VDI"
msgstr "VDI'den SR bulunamadı"

msgid "Unable to get record of VDI"
msgstr "VDI kaydı alınamadı"

msgid "Unable to get updated status"
msgstr "Güncellenen durum alınamıyor"

msgid "Unable to introduce VDI for SR"
msgstr "VDI SR için tanıtılamadı"

msgid "Unable to introduce VDI on SR"
msgstr "VDI SR üzerinde tanıtılamadı"

msgid "Unable to load the virtualization driver"
msgstr "Sanallaştırma sürücüsü yüklenemiyor"

#, python-format
msgid "Unable to migrate VM %(vm_ref)s from %(host)s"
msgstr "VM %(vm_ref)s %(host)s istemcisinden göç ettirilemiyor"

#, python-format
msgid ""
"Unable to obtain RRD XML for VM %(vm_uuid)s with server details: %(server)s."
msgstr ""
"%(server)s sunucu detaylarına sahip VM %(vm_uuid)s için RRD XML elde "
"edilemiyor."

#, python-format
msgid "Unable to parse rrd of %s"
msgstr "%s rrd'si ayrıştırılamadı"

#, python-format
msgid "Unable to retrieve storage policy with name %s"
msgstr "%s isimli depolama ilkesi alınamadı"

#, python-format
msgid "Unable to set up pool: %s."
msgstr "Havuz ayarlanamıyor: %s."

msgid "Unable to unplug VBD"
msgstr "VBD sökülemedi"

msgid "Unexpected build failure, not rescheduling build."
msgstr "Beklenmeyen inşa başarısızlığı, inşa tekrar zamanlanmıyor."

msgid "Unexpected exception in API method"
msgstr "API metodunda beklenmeyen istisna"

#, python-format
msgid "Unknown cell '%(cell_name)s' when trying to update capabilities"
msgstr "Yetenekler güncellenmeye çalışılırken bilinmeye hücre '%(cell_name)s'"

#, python-format
msgid "Unknown cell '%(cell_name)s' when trying to update capacities"
msgstr ""
"Kapasiteler güncellenmeye çalışılırken bilinmeyen hücre '%(cell_name)s'"

#, python-format
msgid "Unknown content in connection_info/access_mode: %s"
msgstr "connection_info/access_mode içinde bilinmeyen içerik: %s"

msgid "Unshelve attempted but an error has occurred"
msgstr "Askıdan alma denendi ama bir hata oluştu"

msgid "Unshelve attempted but vm_state not SHELVED or SHELVED_OFFLOADED"
msgstr ""
"Askıdan almaya çalışıldı ama vm_state SHELVED veya SHELVED_OFFLOADED değil"

msgid "Vifs plug failed"
msgstr "Vifs takma başarısız"

#, python-format
msgid "Volume %s is associated with the instance but no SR was found for it"
msgstr ""
"Mantıksal sürücü %s sunucu ile ilişkilendirilmiş ama kendisi için bir SR "
"bulunamadı"

#, python-format
msgid "Volume is not writable. Please broaden the file permissions. Mount: %s"
msgstr ""
"Mantıksal sürücü yazılabilir değil. Lütfen dosya izinlerinin kapsamını "
"artırın. Bağlama yeri: %s"

#, python-format
msgid ""
"When loading the module %(module_str)s the following error occurred: %(ex)s"
msgstr "%(module_str)s modülü yüklenirken şu hata oluştu: %(ex)s"

msgid ""
"XenAPI is unable to find a Storage Repository to install guest instances on. "
"Please check your configuration (e.g. set a default SR for the pool) and/or "
"configure the flag 'sr_matching_filter'."
msgstr ""
"XenAPI konuk sunucuların yükleneceği Depolama Havuzunu bulamıyor. Lütfen "
"yapılandırmanızı kontrol edin (örn. havuz için varsayılan SR ayarlayın) ve/"
"veya 'sr_matching_filter' bayrağını yapılandırın."

msgid "_migrate_disk_resizing_down failed. Restoring orig vm"
msgstr "_migrate_disk_resizing_down başarısız. Asıl vm geri yükleniyor"

#, python-format
msgid "_migrate_disk_resizing_up failed. Restoring orig vm due_to: %s."
msgstr ""
"_migrate_disk_resizing_up başarısız. Asıl vm geri yükleniyor sebep: %s."

#, python-format
msgid "allocate_port_for_instance returned %(ports)s ports"
msgstr "allocate_port_for_instance %(ports)s bağlantı noktaları döndürdü"

msgid "attaching network adapter failed."
msgstr "ağ bağdaştırıcısı ekleme başarısız."

msgid "detaching network adapter failed."
msgstr "ağ bağdaştırıcısı ayırma başarısız."

msgid "error during stop() in sync_power_state."
msgstr "sync_power_state stop() sırasında hata."

#, python-format
msgid "error opening rbd image %s"
msgstr "%s rbd imajının açılmasında hata"

#, python-format
msgid "kill -HUP dnsmasq threw %s"
msgstr "kill -HUP dnsmasq %s fırlattı"

#, python-format
msgid "killing radvd threw %s"
msgstr "radvd öldürme işlemi %s fırlattı"

msgid "nbd module not loaded"
msgstr "nbd modülü yüklü değil"

msgid "set_admin_password failed"
msgstr "set_admin_password başarısız"

#, python-format
msgid ""
"status is UNKNOWN from vm_state=%(vm_state)s task_state=%(task_state)s. Bad "
"upgrade or db corrupted?"
msgstr ""
"durum BİLİNMİYOR vm_state=%(vm_state)s task_state=%(task_state)s. Kötü "
"yükseltme ya da db bozuk olabilir mi?"
